pixi.toml

```diff
@@ -14,7 +14,7 @@ dev = { depends_on = [
   "dev-install-hydromt",
   "dev-install-pre-commit",
 ] }
-dev-env-file = "python make_env.py full -n hydromt-dev"
+dev-env-file = "python make_env.py full -n hydromt-dev -p 3.9"
```

Comment:
I think this change should not be committed right?

Reply:
I'd like to keep it there, I want to start working on 3.9 again so that I can catch things earlier and this is what I use to recreate my env.